### PR TITLE
NRPT-586 Create NRPTI, LNG, and NRCED build configs

### PR DESCRIPTION
**Pull request by BcGovNeal**
https://bcmines.atlassian.net/browse/NRPT-586

- Added NRPTI, LNG and NRCED build configs
- Removed some unused configs and json configs (replaced by yaml)

### CODE REVIEW

File: tools/angular-on-nginx/admin-nrced.bc.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      triggers:
+        - type: ImageChange
+          imageChange:
+        - type: ConfigChange
+parameters:
+  - description: A name used for all objects
+    displayName: Name
+    name: NAME
+    required: true
+    value: admin-nrced
```
Comment (danieltruong) [inline tools/angular-on-nginx/admin-nrced.bc.yaml:69]: This should actually be nrpti

Comment (BcGovNeal) [reply tools/angular-on-nginx/admin-nrced.bc.yaml:69]: `admin-nrced` is the name of the build, not the image.  The image name is still `nrpti` as discussed.  Changing this value will require changes to the `Jenkinsfile` and then changes to the builds on OCP3.

**Review by mtCarto — APPROVED** [review]

File: tools/angular-on-nginx/admin-nrced.bc.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      completionDeadlineSeconds: 300
+      failedBuildsHistoryLimit: 5
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: nrpti:latest
+      resources:
+        limits:
+          cpu: "2"
```
Comment (MaxWardle) [inline tools/angular-on-nginx/admin-nrced.bc.yaml:33]: are these a bit high?

File: tools/angular-builder/angular-builder.bc.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        build: ${NAME}-build
+    spec:
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}:latest
+      resources:
+        limits:
+          cpu: "2"
```
Comment (MaxWardle) [inline tools/angular-builder/angular-builder.bc.yaml:31]: and this

File: tools/angular-on-nginx/public-lng.bc.yaml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      completionDeadlineSeconds: 300
+      failedBuildsHistoryLimit: 5
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}:latest
+      resources:
+        limits:
+          cpu: "2"
```
Comment (MaxWardle) [inline tools/angular-on-nginx/public-lng.bc.yaml:33]: and here

File: tools/angular-on-nginx/public-nrced.bc.yaml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+      completionDeadlineSeconds: 300
+      failedBuildsHistoryLimit: 5
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}:latest
+      resources:
+        limits:
+          cpu: "2"
```
Comment (MaxWardle) [inline tools/angular-on-nginx/public-nrced.bc.yaml:33]: and here

File: tools/api/api.bc.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      completionDeadlineSeconds: 300
+      failedBuildsHistoryLimit: 5
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}:latest
+      resources:
+        limits:
+          cpu: "2"
```
Comment (MaxWardle) [inline tools/api/api.bc.yaml:33]: and here

File: tools/nginx-runtime/nginx-runtime.bc.yaml [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        app: ${GROUP_NAME}
+    spec:
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}:latest
+      resources:
+      limits:
+        cpu: "2"
```
Comment (MaxWardle) [inline tools/nginx-runtime/nginx-runtime.bc.yaml:29]: and here

**Review by MaxWardle — COMMENTED** [review]
I think it looks good other than the limits and requests 👍

**Review by MaxWardle — APPROVED** [review]
Due to these being build pod requests instead of deployment pod requests, I don't see an issue.

**Pull request merged**